ui/app/templates/components/global-header.hbs

```diff
@@ -10,6 +10,7 @@
       data-test-header-gutter-toggle
       class="gutter-toggle"
       aria-label="menu"
+      role="img"
```

Comment:
Newer versions of our test runner will yell at us if we don't have an appropriate role for items with `aria-label` attributes. Basically the rule is "Spans shouldn't have aria-labels, you can just put text in them!" but this long-existing couple of items have been used as image-replacements and the role is appropriate (this is the hamburger/menu toggle for small screens)